packages/postcss-tape/src/index.ts

```diff
@@ -87,7 +90,13 @@ export default function runner(currentPlugin: PluginCreator<unknown>) {
 
 		// https://github.com/postcss/postcss/blob/main/docs/guidelines/plugin.md#11-clear-name-with-postcss--prefix
 		// Clear name with postcss- prefix
-		if (!packageInfo.name.startsWith('postcss-') && !packageInfo.name.startsWith('@csstools/postcss-')) {
+		const isOlderPackageName = [
+			'css-has-pseudo',
+			'css-blank-pseudo',
+			'css-prefers-color-scheme',
+		].includes(packageInfo.name);
```

Comment:
Should we deprecate and republish?

Reply:
I think very few people use these as a direct dependency.
Looking at the download numbers on npm seems to confirm this.

I think we can deprecate and republish without causing much disruption.

`:has` already has a breaking change planned, and `prefers-color-scheme` could use one to clean up old code.

I'll create an issue for this